packages/site/src/layout/SearchBox.tsx

```diff
@@ -163,6 +166,39 @@ export const SearchBox = ({
                 </ContentCardWrapper>
               </Content>
             )}
+            {filteredComponentList.length == 0 &&
```

Comment:
```suggestion
            {!filteredComponentList.length && !filteredDesignList.length}
```
I think this will do it

Reply:
I know Taylor is doing some refactoring of this component to clean things up so I am just going to leave this as is

Reply:
Misunderstood the proposed change and added it in afterall